modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc

```diff
@@ -20,7 +20,6 @@ The ID is an anchor that links to the module. Avoid changing it after the module
 Start the title of a procedure module with a gerund, such as Creating, Installing, or Deploying.
 ////
 
-[role="_abstract"]
 Write a short introductory paragraph that provides an overview of the module. The introduction should include what the module will help the user do and why it will be beneficial to the user. Include key words that relate to the module to maximize search engine optimization.
```

Comment:
Just wondering if we should add a line with the guideline about leaving a space between the header and the introductory paragraph to avoid any rendering issues? I noticed this today as I was previewing content I was working on myself. And ref Emily's comment at https://github.com/redhat-documentation/modular-docs/pull/185#pullrequestreview-920760221.